docs/source/index.md

```diff
@@ -34,17 +34,24 @@ The hadge pipeline is implemented in Nextflow. To get started, you need to insta
 As next, please run the pipeline
 
 ```bash
-nextflow run http://github.com/theislab/hadge
+nextflow run http://github.com/theislab/hadge -r main
 ```
 
-## **Quick start**
+You can also:
 
 - Choose the mode: `--mode=<genetic/hashing/rescue>`
 - Specify the folder name `--outdir` to save the output files. This will create a folder automatically in the project directory.
 - Specify the input data for each process.
 - The pipeline can be run either locally or on a HPC with different resource specifications. As default, the pipeline will run locally. You can also set the SLURM executor by running the pipeline with `-profile cluster`.
 - Please also check [](usage) for more details.
 
+## **Quick start**
+
+```bash
+sh test_data/download_data.sh
+nextflow run main.nf
```

Comment:
Maybe specify conda or docker here or something? Can you resemble a `test` profile like that the nf-core pipelines have?